packages/lexical-editor/src/plugins/FloatingLinkEditorPlugin/FloatingLinkEditorPlugin.tsx

```diff
@@ -162,6 +163,7 @@ function FloatingLinkEditor({
                         <input
                             type={"checkbox"}
                             checked={linkUrl.target === "_blank"}
+                            disabled={isUrlLinkReference(linkUrl.url)}
```

Comment:
Why is this here? Can we add a comment?

Reply:
You know what.. I just figured it's disabling this checkbox here: 
![image](https://github.com/webiny/webiny-js/assets/5121148/8ac22e8b-8a7e-4b02-a590-410eff3a291a)

Reply:
When we link the elements on the same page, we want the page to scroll to the linked element on the same page.

To prevent the user to open reference links like `<a href="#scroll-to-my-title">link</a>` in another tab, I disabled the checkbox option user to select that possibility.

<img width="356" alt="image" src="https://github.com/webiny/webiny-js/assets/82515066/b5efd84a-f87c-4236-b9a3-b6be5866b016">